stdlib/public/core/Shims.swift

```diff
@@ -36,3 +36,8 @@ internal var _fastEnumerationStorageMutationsTarget: CUnsignedLong = 0
 internal let _fastEnumerationStorageMutationsPtr =
   UnsafeMutablePointer<CUnsignedLong>(Builtin.addressof(&_fastEnumerationStorageMutationsTarget))
 #endif
+
+@usableFromInline
```

Comment:
In addition to being `@usableFromInline`, this should also be `@_alwaysEmitIntoClient`, so that user code doesn't try to link against a nonexistent symbol in older Swift runtime dylibs.

Reply:
Done.

Thanks, that's the piece of knowledge I was missing here.